src/test/regress/sql/multi_mx_modifying_xacts.sql

```diff
@@ -150,12 +150,6 @@ INSERT INTO labs_mx VALUES (6, 'Bell labs_mx');
 SELECT count(*) FROM researchers_mx WHERE lab_id = 6;
 ABORT;
 
--- applies to DDL
-BEGIN;
-INSERT INTO labs_mx VALUES (6, 'Bell labs_mx');
-ALTER TABLE labs_mx ADD COLUMN motto text;
```

Comment:
This now falls back to the `operation is not allowed on this node` messages, which is already covered by other tests.